src/gc/os.d

```diff
@@ -69,13 +109,22 @@ else static if (is(typeof(malloc)))
     version = GC_Use_Alloc_Malloc;
 else static assert(false, "No supported allocation methods available.");
 +/
+/**
+ * Indicates if an implementation support fork().
+ *
+ * The value shown here is just demostrative, the real value is defined based
```

Comment:
I don't think this will end up in any official documentation, so just move the description above to the first definition.